contributor-book/plugins.md

```diff
@@ -467,6 +467,8 @@ Example:
 Nushell rocks!
 ```
 
+For a full example, see [`nu_plugin_example` in the Nushell source repository](https://github.com/nushell/nushell/tree/main/crates/nu_plugin_example).
```

Comment:
We will soon be removing that path. The new path will be https://github.com/nushell/plugin-examples/tree/main/rust/nu_plugin_example

Reply:
```suggestion
For a full example, see [`nu_plugin_example` in the Nushell source repository](https://github.com/nushell/plugin-examples/tree/main/rust/nu_plugin_example).
```